TeadsSDKDemo/app/src/main/java/tv/teads/teadssdkdemo/MainActivity.kt

```diff
@@ -189,7 +189,7 @@ class MainActivity : AppCompatActivity() {
         private const val LOG_TAG = "MainActivity"
         private const val INTENT_EXTRA_PID = "ext_pid"
         const val SHAREDPREF_PID = "sp_pid"
-        const val SHAREDPREF_PID_DEFAULT = 84242
+        const val SHAREDPREF_PID_DEFAULT = 124859
```

Comment:
I am not sure about it, but since we were forcing to use this pid to Direct should I replace it here as I did?

Reply:
Don't change it. We use to inRead. Create a way to change the pid for native/inread without overwrite one or another